Unable to cross compile Zeromq Dependency for arm64-apple-darwin

<!-- Describe the issue -->

Trying to compile starting with dependencies fails with 

checking build system type... x86_64-pc-linux-gnu
checking host system type... Invalid configuration `arm64-apple-darwin': machine `arm64-apple' not recognized
configure: error: /bin/bash config/config.sub arm64-apple-darwin failed

**Expected behavior**

<!--- What behavior did you expect? -->

**Actual behavior**

checking for arm64-apple-darwin-pkg-config... no
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for xmlto... no
checking for asciidoc... no
checking build system type... x86_64-pc-linux-gnu
checking host system type... Invalid configuration `arm64-apple-darwin': machine `arm64-apple' not recognized
configure: error: /bin/bash config/config.sub arm64-apple-darwin failed
make: *** [funcs.mk:289: /home/bitcoin/Downloads/bitcoin-master/depends/work/build/arm64-apple-darwin/zeromq/4.3.4-0f6f30942e0/./.stamp_configured] Error 1


**To reproduce**

command : make HOST=arm64-apple-darwin -j 8

**System information**

Master Branch
Ubuntu 20.04 
Core i5 intel 16GB RAM

Thank you. Finally. Depends built OK, but now running ``` CONFIG_SITE=$PWD/depends/arm64-apple-darwin/share/config.site ./configure ``` fails with 

```
configure: loading site script /home/bitcoin/Downloads/bitcoin-master/depends/arm64-apple-darwin/share/config.site
checking for arm64-apple-darwin-pkg-config... /usr/bin/pkg-config --static
checking pkg-config is at least version 0.9.0... yes
checking build system type... x86_64-pc-linux-gnu
checking host system type... Invalid configuration `arm64-apple-darwin': machine `arm64-apple' not recognized
configure: error: /bin/bash build-aux/config.sub arm64-apple-darwin failed
make: *** No targets specified and no makefile found.  Stop.

```

--- a/depends/packages/zeromq.mk
+++ b/depends/packages/zeromq.mk
@@ -20,12 +20,12 @@ endef
 
 define $(package)_preprocess_cmds
   patch -p1 < $($(package)_patch_dir)/remove_libstd_link.patch && \
-  patch -p1 < $($(package)_patch_dir)/netbsd_kevent_void.patch && \
-  cp -f $(BASEDIR)/config.guess $(BASEDIR)/config.sub config
+  patch -p1 < $($(package)_patch_dir)/netbsd_kevent_void.patch
 endef
 
 define $(package)_config_cmds
   ./autogen.sh && \
+  cp -f $(BASEDIR)/config.guess $(BASEDIR)/config.sub config && \
   $($(package)_autoconf)
 endef
 

